### DISCUSSION TITLE
What happened to the "column privileges" tab on Supabase Studio?

### DISCUSSION THREAD

**Discussion by joaoguidev** [supabase/supabase#21713] [Questions] [answered]
Hello.

Going through Supabase documentation I saw that at some point there was a "Column Privileges" were it was possible to see what privileges each role had on every table.  This was on alpha as shown in the [documentation ](https://supabase.com/docs/guides/auth/column-level-security#manage-column-privileges-in-the-dashboard) and it seems that it was removed, at least I couldn't find it.

**That was a lovely idea.** 

Is this still around or are there plans to have it back in the future?

Thank you.

**Comment by GaryAustin1** [comment, marked as answer]
Go to the person icon at bottom of dashboard.  See Feature Previews.
![image](https://github.com/supabase/supabase/assets/54564956/c0c1bdd8-6bcc-4eae-8291-eb469aae8aff)

Note warnings before using.
